test/cli.js

```diff
@@ -31,6 +31,17 @@ test.failing('ignores fixture', async t => {
 	t.throws(execa('../../../cli.js', ['--no-local'], {cwd}));
 });
 
+test('ignore files in .gitignore', async t => {
```

Comment:
Would be good if this test was duplicated as an API test too (using the `cwd` option).